### PR TITLE
Bump open-aea to v1.19.0

### PR DESCRIPTION
**Pull request by angrybayblade**
## Proposed changes

TBD

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -233,11 +233,17 @@ fix-abci-app-specs:
 	echo "Successfully validated abcis!"
 
 
-AEA_AGENT_HELLO_WORLD:=valory/hello_world:latest:$(shell cat packages/hashes.csv | grep "agents/hello_world" | cut -d "," -f2 )
-AEA_AGENT_ORACLE:=valory/oracle:latest:$(shell cat packages/hashes.csv | grep "agents/oracle" | cut -d "," -f2 )
-AEA_AGENT_APY_ESTIMATION:=valory/apy_estimation:latest:$(shell cat packages/hashes.csv | grep "agents/apy_estimation," | cut -d "," -f2 )
+AEA_AGENT_HELLO_WORLD:=valory/hello_world:latest:$(shell cat packages/packages.json | grep "agents/hello_world" | cut -d "," -f2 )
```
Comment (DavidMinarsch) [inline Makefile:236]: does this command still work?

File: Makefile [hunk 2]
```diff
@@ -233,11 +233,17 @@ fix-abci-app-specs:
...
 	export AEA_AGENT_ORACLE=${AEA_AGENT_ORACLE}
 	export AEA_AGENT_APY_ESTIMATION=${AEA_AGENT_APY_ESTIMATION}
 	export AEA_AGENT=${AEA_AGENT_HELLO_WORLD}
 	skaffold build -p release --cache-artifacts=false && skaffold build -p release-latest
+
+
+.PHONY: hashes
+hashes:
+	autonomy hash all
+	autonomy packages lock
```
Comment (DavidMinarsch) [inline Makefile:249]: we have `generators` for this. Please revert

File: autonomy/cli/hash.py [hunk 3]
```diff
@@ -129,8 +126,6 @@ def update_hashes(  # pylint: disable=too-many-locals
                 if vendor is not None and package_id.author != vendor:
                     continue  # pragma: nocover
                 package_hashes[key] = package_hash
-
-        to_csv(package_hashes, packages_dir / HASHES_FILE)
```
Comment (DavidMinarsch) [inline autonomy/cli/hash.py:133]: So the update won't take effect? where is it being saved?

**Review by DavidMinarsch — COMMENTED** [review]
You need to run `make generators`

File: plugins/aea-test-autonomy/setup.py [hunk 4]
```diff
@@ -47,7 +47,7 @@ def _get_docker_dependency() -> str:
 
 
 base_deps = [
-    "open-aea[all]>=1.18.0.post2,<2.0.0",
+    "open-aea[all]>=1.19.0,<2.0.0",
     "pytest==7.0.0",
     "open-aea-ledger-ethereum==1.18.0",
```
Comment (Karrenbelt) [inline plugins/aea-test-autonomy/setup.py:52]: `"open-aea-ledger-ethereum==1.19.0"` ?

**Review by Karrenbelt — CHANGES_REQUESTED** [review]
where `hashes.csv` can still be found:

https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/autonomy/cli/hash.py#L75

bunch here (there's more):
https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/scripts/check_doc_ipfs_hashes.py#L56
https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/scripts/check_doc_ipfs_hashes.py#L81

this one seems important:
https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/scripts/check_ipfs_hashes_pushed.py#L76

**Comment by DavidMinarsch** [comment]
> where `hashes.csv` can still be found:
> 
> https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/autonomy/cli/hash.py#L75
> 
> bunch here (there's more in this file):
> 
> https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/scripts/check_doc_ipfs_hashes.py#L56
> 
> 
> https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/scripts/check_doc_ipfs_hashes.py#L81
> 
> this one seems important:
> 
> https://github.com/valory-xyz/open-autonomy/blob/049bd3deb145bc9303b662d0553420bf9f6de40c/scripts/check_ipfs_hashes_pushed.py#L76

Re the last: https://github.com/valory-xyz/open-aea/issues/313 -> can be done after release in interest of time

File: autonomy/cli/hash.py (hunk 3 above)
Comment (DavidMinarsch) [reply autonomy/cli/hash.py:133]: @angrybayblade

Comment (angrybayblade) [reply autonomy/cli/hash.py:133]: Won't be saved, will be done on `autonomy packages lock`

**Review by DavidMinarsch — DISMISSED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**